meeting-notes/2019-12-2_metadata-with-ramona.md

```diff
@@ -0,0 +1,36 @@
+# Metadata Meeting with Ramona
+### 2019-12-2
+
+Present: Emily Cain, Ramona Walls
+
+### Notes
+
+* Discussed and shared the Cornell best practices for writing "readme" style [metadata](https://data.research.cornell.edu/content/readme)
+* Metadata should be human-readable (like the Cornell template) and also computer-readable
+    * JSON / XML / [EML](https://www.dataone.org/software-tools/tags/EML)
+    * The University Library may not require computer-readable metadata because they want to make it easy for users to publish data, but some researchers use web scrapers to find data, so being deliberate about using keywords in the description is extremely important.
+    * Additional metadata above and beyond the basic requirements for a DOI is ideal.
+    * Ramona will be re-learning EML so can be a resource in this area, in addition to Jeanette.
+* Reproducing code and data transformations
+    * Executable Jupyter Notebook ([Binder](https://mybinder.org/) button would be great)
+    * `.py` script
+* Every dataset / data file needs to be described
+    * If image datasets are large and variegated, for example, people should be able to pinpoint and find exactly what they need. 
```

Comment:
"large and varied" perhaps?

Reply:
Thanks for reviewing this! Yes, that is a much better word in this non-botanical context.